templates/index.js

```diff
@@ -0,0 +1,21 @@
+function formatCrateName(crateTitleAndVersion) {
+    const stringParts = crateTitleAndVersion.split(" ", 2);
+    return stringParts[0] + ' = "' + stringParts[1] + '"';
+}
+
+function copyTextHandler() {
+    const crateTitleAndVersion = document.getElementById("crate-title").innerText;
+    const temporaryInput = document.createElement("input");
+
+    temporaryInput.type = "text";
+    temporaryInput.value = formatCrateName(crateTitleAndVersion);
+
+    document.body.append(temporaryInput);
+    temporaryInput.select();
+    document.execCommand("copy");
+
+    temporaryInput.remove();
+}
+
+document.getElementById("clipboard")
+    .addEventListener("click", copyTextHandler);
```

Comment:
It doesn't fit in 100 columns?

Reply:
Oh that was left over from before I refactored some things; it should be on its own line. Thanks for the catch!